vivisect/parsers/pe.py

```diff
@@ -339,6 +339,11 @@ def loadPeIntoWorkspace(vw, pe, filename=None, baseaddr=None):
     exports = pe.getExports()
     for rva, ord, name in exports:
         eva = rva + baseaddr
+
+        # Functions exported by ordinal only have no name
+        if name is None:
```

Comment:
just in case, can you do `if not name`, just in case the name is the empty string?

Reply:
done!